src/Controller/UI/ReportFundingOrganizationController.php

```diff
@@ -81,7 +82,7 @@ private function createCsvReportFileName(string $fundingOrgName, string $funding
     {
         $nowDateTimeString = date(self::REPORTFILENAMEDATETIMEFORMAT);
 
-        $researchGroupNameSubstring = substr($fundingOrgName, 0, self::MAXRESEARCHGROUPLENGTH);
+        $researchGroupNameSubstring = substr($fundingOrgName, 0, self::MAXFUNDINGORGLENGTH);
```

Comment:
Where is this used? Also the variable name is confusing?

Reply:
Not used, copy paste code and i don't know why i made changes to that. Fixed it now